qa/common/shm_util.py

```diff
@@ -434,12 +435,14 @@ def __exit__(self, type, value, traceback):
             for current_shm_size, prev_shm_size in zip(
                 current_shm_sizes, self._shm_region_free_sizes
             ):
-                if current_shm_size != prev_shm_size:
+                if current_shm_size > prev_shm_size:
```

Comment:
@rmccorm4 Do you think the comparison should be `<` instead of `>`? I think the `*_shm_size` here is not referring to the occupied size, but the free size on the shm segment. If there is a leak, then `current_free_size < previous_free_size`.